### PR TITLE
Add blog post for Podman 1.0 release

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

File: _posts/2019-1-16-podman-release-v1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+tags: community, open source, podman
+---
+
+# Podman Release 1.0.0
+
+![podman logo](https://podman.io/images/podman.svg)
+
+## Podman has gone 1.0!
+
+Our original goal with Podman was to provide a fully-featured debugging experience for [CRI-O](https://github.com/kubernetes-sigs/cri-o), but it has become so much more. Podman 1.0.0 is a fully-featured container engine.  It provides a Docker-compatible command line to ease the transition from other container engines. Most Podman commands can be run as a regular user, without requiring additional privileges. Furthermore, all of this is accomplished without a daemon!
```
Comment (TomSweeneyRedHat) [inline _posts/2019-1-16-podman-release-v1.0.0.md:15]: I'm waivering.  Should we have the link to CRI-O as you have it, or https://cri-o.io/?  @rhatdan thoughts?

File: _posts/2019-1-16-podman-release-v1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Finally, the Podman team would like to thank all our contributors. Everyone who submitted code, improved documentation, or reported bugs has been a great help.
+
+## Changes
+A few of the biggest changes from Podman 1.0.0 include:
+
+    * Added the `podman play kube` command, which creates Podman pods based on Kubernetes pod YAML.
+    * The `podman run` and `podman create` commands now support the `--init` flag, to run a minimal init process in the container.
+    * Added the `podman image sign` command to sign container images.
+    * Image pulls are now parallelized for increased speed
```
Comment (TomSweeneyRedHat) [inline _posts/2019-1-16-podman-release-v1.0.0.md]: For reasons I'm not understanding/seeing.  These bulleted items are in a scroll box in the preview.  Is it showing that way for you?  I'd prefer having them in an aligned bulleted list if possible.

Comment (mheon) [reply _posts/2019-1-16-podman-release-v1.0.0.md]: Pushed a fix

**Comment by TomSweeneyRedHat** [comment]
LGTM
To kill the scroll bar, did you remove some spaces before the '*'?
@mheon @rhatdan gtg when you want to pull the trigger.

**Comment by mheon** [comment]
@TomSweeneyRedHat Yeah - 4 spaces gets a gray background/scroll bar, 2 spaces is just bullet points

**Comment by mheon** [comment]
@rhatdan We good to merge? Want to get this out before end of day

**Pull request merged**